go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/spf13/cobra v1.1.3
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.6.1
+	golang.org/x/mod v0.4.1 // indirect
```

Comment:
We should add a verify-go-mod script because this isn't `indirect` any more and `go mod tidy` fixes that.